sympy/geometry/ellipse.py

```diff
@@ -28,7 +28,7 @@
 from .point import Point, Point2D, Point3D
 from .line import Line, LinearEntity
 from .util import idiff
-
+from sympy.core.symbol import var
```

Comment:
I think you should use relative imports, and not `var`, but `symbols`, see below.